src/features/components/atoms/table/base/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactNode } from 'react'
+import Box, { BoxProps } from '../../box'
+import { Localize } from 'components/localization'
+import { TString } from 'types/generics'
+import dclsx from 'features/utils/dclsx'
+import { ClassProps } from 'features/types'
+
+type Data = Record<string, string | ReactNode>
+
+export interface TableProps extends BoxProps, ClassProps {
+    striped?: boolean
+    data: Data[]
+}
+
+const BaseTable = ({ data, striped, className, ...rest }: TableProps) => {
+    return (
+        <Box
+            className={dclsx('table-responsive', { ['table-striped']: striped }, className)}
+            {...rest}
+        >
+            <table className={dclsx('table')}>
+                <thead>
+                    <tr>
+                        {Object.keys(data[0]).map((key) => (
```

Comment:
better to split the data structure, as column data, and row data  so we can avoid these kinds of complex mappings 
just a suggestion

Reply:
Done..